packages/product/integration-tests/__tests__/module.ts

```diff
@@ -40,6 +51,12 @@ describe("Product module", function () {
       expect(module).toBeDefined()
     })
 
+    it("should have a connection that is not the shared connection", async () => {
```

Comment:
I don't quite understand this test. Why would the `sharedPgConnection` be known to the module when its not connecting it anywhere?

Reply:
Just ensuring that we have a shared connection when it should be there and we dont have it when it is not expected. Just to prevent regression